include/mbedtls/sha256.h

```diff
@@ -182,12 +197,13 @@ MBEDTLS_DEPRECATED static inline void mbedtls_sha256_finish(
 }
 
 /**
- * \brief          SHA-256 process data block (internal use only)
+ * \brief          This function processes a single data block within
+ *                 the ongoing SHA-256 computation.
```

Comment:
Blocker: this function is for internal use only. Please keep this explicit.

Reply:
fixed